session.go

```diff
@@ -1995,6 +2001,7 @@ type QueryObserver interface {
 type ObservedBatch struct {
 	Keyspace   string
 	Statements []string
+	Values     [][]interface{}
```

Comment:
Could you please add documentation for the field?

For example:

```go
// Values holds a slice of bound values for each statement.
// Values[i] are bound values passed to Statements[i].
// Do not modify the values here, they are shared with multiple goroutines.
Values     [][]interface{}
```

Reply:
Documentation is added. Thanks